docs/howto/migrate-proxy.md

```diff
@@ -0,0 +1,13 @@
+#### Migrating proxy
+For Migrating Proxy we have to perform some manual steps as well. This document will list all the manual as well as automated process.
+1. Add the server in inventory and assign it the `proxy ` group and `cas_proxy`(For ICDS)
+2. Run  `commcare-cloud <env> ap deploy_shared_dir.yml --tags=nfs --limit=shared_dir_host`
+3. Run `commcare-cloud <env> ansible-playbook letsencrypt_cert.yml`
+4. Run `deploy-stack` for the server.
+5. Do a deploy `cchq <env> deploy`
+3. Copy static content from live proxy `/home/cchq/www/<env>/current/staticfiles` to new server
+4. Copy Letsencrypt Configuration dir from live proxy `/etc/letsencrypt` to new server
+5. Perfrom a dry run for cert check ` certbot renew --dry-run`
```

Comment:
why is it useful to run this?

Reply:
For checking any issue with Letsencrypt.

Reply:
ok, but is it able to do that even though the traffic isn't being routed to this node yet?

Reply:
Moved to post validation section.